\chapter{HIV Neutralizing Antibodies in HIV-Na\"{i}ve Donors}
\label{chap:chapter3}
\section{Introduction}
The induction of broadly neutralizing HIV-specific antibodies is likely to be a critical component of the mechanisms of protection for an effective HIV vaccine. In the work presented here, I used a novel approach for examining the heavy chain complementarity determining region 3 (HCDR3) repertoire of HIV-\naive donors to interrogate the structural properties of long HCDR3 loops. Some broadly neutralizing HIV-neutralizing human antibodies possess long HCDR3 regions, which typically are created at the time of original antibody gene recombination rather than during somatic hypermutation. I sought to determine if antibodies with long, structured HCDR3s that are present in the \naive B-cell repertoire of HIV-\naive donors confer neutralizing properties to those antibodies similar to that of previously isolated HIV-neutralizing antibodies such as PG9 and PG16, which target the HIV envelope protein variable loops 1 and 2. Using ultra-deep nucleotide sequence analysis of HCDR3 regions in antibody gene replicons for 64 different HIV-\naive donors, I obtained approximately 25,000 unique sequences that encoded HCDR3s of 30 amino acids in length, the same size as broadly neutralizing antibody PG9. The modeling suite \rosetta was used to assess the ability of these 30 amino acid length HCDR3 sequences to form a loop with structure similar to that of antibody PG9. The PG9 backbone template then was threaded rapidly with sequences from HIV-\naive donors to evaluate the energetic state of naturally occurring long HCDR3s when tested in a PG9-like conformation. The sequences encoding HCDR3s with the most favorable predicted energy states in this conformation then were redesigned \silico to optimize binding with minimal changes, simulating the process of somatic hypermutation. The sequences that were found to mimic the binding energy and HCDR3 structure of PG9 were synthesized and characterized experimentally for ability to bind to HIV envelope (Env) protein and neutralize HIV infectious virions. I found 12 unique antibodies that present PG9 HCDR3 mimicry with 0 to 7 mutations away from their respective HIV-\naive wild-type sequence. This work, using a robust new bioinformatics and modeling pipeline, suggests that HIV-\naive donors may possess \naive B-cells encoding antibodies with long HCDR3s that can neutralize HIV prior to infection. Expanding and preserving these unique \naive B-cells from the \naive repertoire may represent an important new strategy for HIV vaccine priming.

\subsection{Potential Paradigm Shifts in Vaccinology}
Elicitation of broadly neutralizing antibodies (bNAbs) against human immunodeficiency virus type 1 (HIV-1) has been one of the greatest challenges in modern vaccinology \citep{Ackerman:2013fc}. A bNAb response occurs in 10-30\% of those infected, with about 1\% of those possessing "elite" neutralization breadth \citep{Simek:2009cn}. These antibodies typically arise 1 year after infection at the earliest and peak at approximately 3-4 years after infection \citep{vanGils:2009if}. Recent advances in donor selection from cohorts of chronically infected patients, novel screening methods, and antibody isolation technologies, have allowed identification of dozens of new antibodies to the envelope protein gp120/gp41 of HIV-1 \citep{Kwong:2012cc}. These antibodies bind at least five major epitopes on gp120 including the CD4 binding site \citep{Wu:2011jf}, the gp120 outer domain \citep{Trkola:1996vb},  the V3 loop and V3 complex glycans \citep{Mouquet:2012ka}, the V1/V2 loop and surrounding glycans \citep{Walker:2009cd}, or the membrane proximal region (MPER) on gp41 \citep{Huang:2012fh}. While the various bNAbs of interest were isolated using different strategies, they often share certain unusual and distinctive genetic features, such as very large numbers of somatic mutations or very long heavy chain complimentarity determining region 3 (HCDR3) structures \citep{Corti:2013ex}.

Although studies based on bNAb isolation have revealed new targets for immunogen design and have been useful for experimental therapeutic efficacy \citep{Klein:2012gi}, the design of a vaccine against HIV-1 that induces sterilizing or protective immunity has remained elusive \citep{McCoy:2013jq}. For example, the RV144 trial that tested a vaccine strategy using priming immunization with a recombinant canarypox vector followed by a bivalent gp120 protein boosting immunization revealed 31\% efficacy in the first year for this regimen \citep{RerksNgarm:2009bk}. While the RV144 trial results did suggest modest efficacy, the waning of protection after one year suggested major improvement in immunogenicity is still needed. An obstacle in vaccine development for elicitation of bNAbs is the limited understanding of the maturation process from germline to hypermutated variable gene segments or the process for eliciting secretion of antibodies with long HCDR3s, which are characteristic of a number of mature bNAbs \citep{Haynes:2012ba}.

One recent study conducted by Doria-Rose and colleagues at the Vaccine Research Center (VRC) structurally characterized twelve somatically related V1/V2 binding mAbs that share characteristics with PG9 and PG16, including a long protruding HCDR3 loop that are tyrosine sulfated \citep{DoriaRose:2014ic}. They found that the unmutated common ancestor was present at 30-38 weeks post-infection and was able to weakly neutralize superinfecting virus. This finding is a great step in corroborating my rationale for eliciting these types of V1/V2 binding mAbs as a strategy for vaccine development.

Here, I attempted to study the occurrence of antibodies in the \naive B-cell repertoire with long HCDR3s that might mediate neutralization of HIV, prior to exposure to HIV antigens. I examined the HCDR3 repertoire of a large number of HIV-\naive donors in context of the HCDR3 structure of the V2/V3-binding antibody PG9. This antibody was isolated initially from a donor in the IAVI African Protocol G cohort using high-throughput B-Cell microneutralization assays to identify functional B-Cell clones \citep{Walker:2009cd}. These antibodies possess unusual HCDR3 structures characterized by a 30 amino acid long (IMGT numbering) hammerhead-structured HCDR3 region that protrudes about 20$/AA$ from the surface of the rest of the combining site \citep{Pancera:2010hh,Pejchal:2010fp,McLellan:2011dg}. Initial studies using point mutations identified the HCDR3 as the primary structural feature that mediated neutralization \citep{Pancera:2010hh}, and this finding was confirmed later by atomic-resolution structural studies of the mAbs in complex with V1/V2 scaffolds \citep{McLellan:2011dg,Pancera:2013ev}.

Using ultra-deep sequencing technology along with a robust molecular modeling pipeline, I identified long HCDR3 sequences from the B-cells of HIV-\naive donors that were predicted to mimic the unusual structure of mAb PG9. First, I identified 30 amino acid length HCDR3 sequences, which matched the amino length of the PG9 HCDR3. Using pilot comparative modeling experiments in the software package \rosetta \citep{LeaverFay:2011fc,Kaufmann:2010ea}, I created position specific structure scoring matrices (P3SMs) to score the large number of HCDR3 sequences returned from HCDR3 sequencing rapidly without the need for a full molecular modeling trajectory to assess the ability of the sequences to achieve a PG9-like loop structure. The sequences were ranked by predicted tolerance for assuming the structure of the HCDR3 of PG9 in the PG9/CAP45 complex. Sequences that returned scores close to those of PG9 were characterized experimentally by synthesis and expression of recombinant antibodies, followed by testing for binding to HIV antigens and neutralization of HIV. I found 12 antibodies that mimics the function of PG9 with at least one mAb that neutralized HIV.

Here, I used a structural convergence paradigm, as I found the sequences that mimicked PG9 function by searching for antibodies with the ability to form a similar structure, rather than clones with a high level of genetic identity or homology. The results suggest that a vaccine strategy based on elicitation of PG9-like antibodies may provide several advantages in rational vaccine design for HIV, since PG9-like HCDR3 loops are found in the HCDR3 repertoire of HIV-\naive individuals. Furthermore, the number of somatic mutations necessary to achieve broad and potent neutralization is less than the extraordinary number of mutations typical of many conventional bNAbs \citep{Klein:2013iz}. Taken together, the targeting of B-Cell receptors with sequences that encode PG9-like HCDR3 structures in the repertoire of HIV-\naive individuals may provide a critical target in immunogen design for HIV vaccine development.

\subsection{Ultra High-Throughput Sequencing}
Through the remainder of this section I will refer to next-generation sequencing as high-throughput sequencing (HTS). These can be divided into technologies that often produce a smaller number of reads but allow a much longer read length. This approach includes Roche 454-pyrosequencing~\textcopyright{}, Illumina MiSeq~\textcopyright{},  IonTorrent~\textcopyright{}, and PacBio~\textcopyright{} platforms. These high-throughput sequencing technologies give a throughput from 0.7 to 95 gigabases (figure \ref{fig:figure3_1} and table \ref{tab:table3_1})  \citep{developmentinNGS:2012bs}. In contrast, some platforms fall into what I call ultra high-throughput sequencing (UHTS), including those of the Illumina HiSeq~\textcopyright{} and SoLiD~\textcopyright{} platform, which tends to give shorter reads but achieves higher number of reads, from 320-600 gigabases (figure \ref{fig:figure3_1}, table \ref{tab:table3_1}). The advances in HTS spurred an initial investigation to examine the healthy donor repertoire in my laboratory.

\begin{figure}
   \centering
   \includegraphics[scale=.7]{images/chapter3/figure3_1.pdf} % requires the graphicx package
   \caption[Current Sequencing Technologies]{Current sequencing technologies. On the x-axis is the current read length for each sequencing platform. The y-axis is the bases per run. Each point is a new iteration of that platforms sequencing read length and coverage. HiSeq has the most coverage with relatively short read lengths. Figure adapted from  \citep{developmentinNGS:2012bs} }
   \label{fig:figure3_1}
\end{figure}
\input{tables/chapter3/table3_1.tex}

\subsection{Long HCDR3s are Established at Original Recombination}
Using first generation HTS technology, I first observed long HCDR3 sequences in the healthy HIV-\naive repertoire \citep{Briney:2012ib}. Although long HCDR3s were known to exist \citep{Zemlin:2003jn,Ivanov:2005uu} they are primarily found in chronically HIV-infected subjects \citep{Walker:2009cd,Spurrier:2011ia,Choe:2003un,Walker:2011ew}. This led to a multiple hypothesis model about the origin of long HCDR3s in chronically infected patients (figure \ref{fig:figure3_2}). Given that long HCDR3 antibodies against HIV are highly mutated relative to other antibodies, it could be hypothesized that insertions and deletions (indels) due to somatic mutations could be responsible for the `elongation' of HCDR3s. These antibodies could be elicited by a chronic infection or heterogeneous prime-boost vaccine strategy, both of which will exhibit selective pressure for antibody maturation from germline sequence (figure \ref{fig:figure3_2}). This long HCDR3 origin model was known as the `mutational' model.The alternate hypothesis was that long HCDR3s were established in the bone marrow at the time of original recombination and were found in the circulating repertoire at low frequency, or were selected against due to known issues associated with autoimmunity \citep{Aguilera:2001uo,Wardemann:2003km,Ichiyoshi:1994wb,Crouzier:1995ua}. This is known as the "original recombination" model.

These two models for the origin of long HCDR3s were the original focus for this project and are detailed in specific aims of the 2012 thesis of Bryan Briney from the Crowe Laboratory. The models were tested by comparing immunoglobulin sequence repertoires for long HCDR3s to those of canonical length HCDR3s \citep{Briney:2012tg}. I can further test these models based on two main assumptions about antibody maturation process. One, all antibodies are generated in the bone marrow from the germline repertoire and affinity mature in the lymph tissue \citep{Tonegawa:1983vw,Murphy:2007tl}. Thus, an antibody that is recently recombined in the bone marrow would have fewer mutations than antibodies that have gone through multiple rounds of affinity maturation.  Figure \ref{fig:figure3_4} A (top panel) shows an analysis of peripheral B-cell mean number of mutations as a function of HCDR3 length. Longer HCDR3s tend to have lower levels of somatic mutations. Figure \ref{fig:figure3_3} B (top panel) specifically shows the correlation for IgM and IgG class types that have undergone positive and negative selection.

The second assumption is that the affinity maturation process is associated with insertion and deletion events that are likely to be caused by somatic hypermutation associated proteins \citep{Reason:2006cp,Wilson:1998wo,Wilson:1998ut}. Figure \ref{fig:figure3_4} A and \ref{fig:figure3_4} B (lower panels) show a negative correlation to insertion-deletion associated (indel) percentage,suggesting further evidence supporting the `original recombination' model. Finally, a statistically significant change in long HCDR3 percentage between \naive B-cells and class switched IgM and IgG B-cells indicates that these antibody sequences are present at original recombination and are selected against during maturation.

Although the original goal for the work done by Bryan Briney was to determine the genetic origin of HCDR3s, it confirmed a population, albeit a very small one, of very long HCDR3 sequences that are the same length as many of the exceptional broadly neutralizing antibodies against HIV-1. This fortuitous conclusion, along with rapidly advancing throughput of HTS technology, led to the experimental rationale.

\begin{figure}[!tH]
   \centering
   \includegraphics[width=.99\textwidth]{images/chapter3/figure3_2.pdf} % requires the graphicx package
   \caption[Origins of Long HCDR3 Models]{Origins of long HCDR3 models. Two models are proposed to explain the origin of HCDR3s. In the mutation model (left), B-cells (yellow) with canonical length HCDR3s are `elongated' through selective pressure on B-cells (blue) by chronic infection or repeated rounds of vaccine boosts to trigger affinity maturation to an evolving antigen. This repeated round of exposure creates insertions into the long HCDR3. The original recombination model (left) assumes the long HCDR3 is in low frequency in the \naive population (yellow). Antigens from a vaccine or chronic infection select the infrequent B-cell and expand its clonal population and refine affinity.}
   \label{fig:figure3_2}
\end{figure}


\subsection{Rationale and Experimental Design}
My conclusions from examining the HIV-\naive donor repertoires made me consider various existing approaches to HIV vaccine design. Traditional vaccinology assumes that one can make a germline antibody into a broadly neutralizing antibody by a series of prime-boost steps to achieve an optimal level of mutations to produce a protective response to HIV challenge. Indeed, many great strides have been made in the field \citep{Liao:2013gs,JavierGuenaga:2011bf,Kwong:2009hp,DoriaRose:2012if}. However, the other unusual characteristics about broadly neutralizing antibodies to HIV is many of them possess very long HCDR3s, where almost all of the contact and therefore mechanism of neutralization, stem from the long HCDR3 \citep{McLellan:2011dg,Pejchal:2010fp,Klein:2013iz,Zhou:2007bz}.

With knowledge that long HCDR3s are present in HIV-\naive donors, and that they are established at the time or recombination, I hypothesized that it was possible for these long HCDR3 sequences to mimic some of the long HCDR3-driven broadly neutralizing antibodies. I choose the complex of PG9 and a scaffolded epitope V1/V2 to be my template \citep{McLellan:2011dg}. The goal was to see if I could mimic the 30-length HCDR3 in PG9 to neutralize HIV (PG9 mimicry). I chose PG9 with the following rationale:

\begin{enumerate}
    \item  The co-crystal structure had recently been elucidated (figure \ref{fig:figure3_3}) \citep{McLellan:2011dg}.
	\item  The long HCDR3 accounts for neutralization and functionality with few contact residues in other regions of the antibody \citep{Pancera:2010hh,Pejchal:2010fp}.
	\item  Variants with germline reversions of the framework still retains neutralization ability \citep{Klein:2013iz}.
	\item  The RV144 trial, the first trial to show substantial vaccine efficacy, revealed an increase in V1/V2 binding antibodies (the binding region of PG9) \citep{Haynes:2012ka}.
	\item  Interactions with the V1/V2 region are structure-dependent and sequence independent, \textit{i.e.} backbone-backbone hydrogen bonding across beta-sheets \citep{McLellan:2011dg}.
	\item  There is a 9 mutation difference between PG9 and its sister antibody PG16 at the HCDR3 paratope, showing a structural as well as functional convergence irrespective of sequence similarity \citep{Walker:2009cd,Pancera:2010hh,Pejchal:2010fp}.
\end{enumerate}

Bryan Briney's work with first generation high-throughput sequencing yielded a low frequency of sequences with a length of 30 amino acids (0.4\%). Building on this observation, figure \ref{fig:figure3_5} proposes an experimental plan that could test a large number of sequences for PG9 mimicry. In order to make the proposal feasible, I needed to increase the number of donors sequenced and the amount of B-cell coverage in a new round of next-generation high-throughput sequencing.  I also knew that the coverage necessary was going to produce 500 million to 1 billion sequences, far more data than could be supported on my existing data analysis architecture.

In collaboration with Jessica Finn who did the sequencing, I planned to implement custom databases and search algorithms to collect the sequences into a functional antibodyome. For prediction of PG9 mimicry, I also needed to develop a structural prediction scheme based on the package \rosetta that would rapidly be able to predict whether a sequence could tolerate the PG9 configuration. I also planned for a reasonable number of recombinant antibodies to be made in the laboratory to test for experimental validation using binding and neutralization analysis.

\begin{figure}
   \centering
   \includegraphics[scale=.9]{images/chapter3/figure3_3.pdf} % requires the graphicx package
   \caption[PG9 Complexed with V1/V2 Scaffold]{PG9 in complex with V1/V2 scaffold. The crystal structure adapted from  \citep{McLellan:2011dg}. Beta-sheet interactions at the interface are highlighted.}
   \label{fig:figure3_3}
\end{figure}


\begin{figure}[!t]
   \centering
   \includegraphics{images/chapter3/figure3_4.pdf} % requires the graphicx package
   \caption[Maturation Sequence Markers and HCDR3 Length]{Maturation sequence markers and HCDR3 length. Insertion and deletion (indel) frequency and mutation frequency is associated with affinity maturation and shows a negative correlation with HCDR3 length (A). The correlation becomes more pronounced in peripheral blood classed-switched B cells, IgM and IgG (B). The percentage of the repertoire with long HCDR3s (>24 amino acids, C) and very long HCDR3s (>28 amino acids, D) shows a statistically significant change from \naive B cells to class-switched B cells (*** -- p < 0.001, * -- p < 0.1).}
   \label{fig:figure3_4}
\end{figure}

\begin{figure}[!t]
   \centering
   \includegraphics[width=.9\linewidth]{images/chapter3/figure3_5.pdf} % requires the graphicx package
   \caption[Overview of Methodology ]{The methodology can be split into four subsections that are a combination of computational (dashed-line) and experimental work (solid-line). HIV-\naive donor blood is collected from 64 adult donors and the HCDR3 is sequenced on the HiSeq Illumina platform (A). The raw sequences are reconstructed and analyzed against germline databases using custom software. The sequences are parsed and stored in optimized databases to handle the large quantity of antibody sequences (B). HCDR3 sequences are chosen by length and tested for PG9 mimicry using the \rosetta software suite. Iterative rounds of minimization, docking, and design, followed by rigorous statistical analysis allows for a robust prediction of potential candidates from the HIV-\naive donor repertoire that may neutralize HIV (C). A tractable number of sequences are synthesized and tested experimentally through biophysical characterization and neutralization studies against HIV-1 (D).}
   \label{fig:figure3_5}
\end{figure}

\section{Ultra High-Throughput Sequencing of HCDR3s}
As explained in the rationale, it was necessary to obtain many more sequences than were available using the coverage of 454-pyrosequencing. I decided to use the HiSeq platform, as the throughput at the time was 109 reads of 150 base pairs (bp). Although, the sequencing read of 150 bp was not long enough to cover the entire antibody variable gene, it would provide coverage of the HCDR3 portion of a recombined gene, sufficient for my experimental goals. I estimated that 0.4\% of the sequences would be greater than 28-length at 3 standard deviations from the mean HCDR3 length. This approach in theory, would deliver 400,000 recombined very-long HCDR3 reads.

Using the scheme found in figure \ref{fig:figure3_6}, I indexed antibody gene repertoires from 64 different healthy donors with indexes based on primer design by Bryan Briney and Jessica Finn in the Crowe laboratory. First, I obtained white blood cells from 64 HIV-\naive donors through the American Red Cross. No further information was obtained about each donor except for hepatitis B, C and HIV negative results. The mRNA was purified from the peripheral blood mononuclear cells (PBMC), and subjected to an RT-PCR and PCR. The HiSeq run was done in the VANderbilt Technologies for Advanced GEnomics (VANTAGE). The raw data was reconstructed with paired-end algorithms and run through PyIg (unpublished). This program called on V, D, and J gene segments for the HCDR3 region and stored them to a database custom built for large amounts of information. The statistics for the HiSeq run are found in table \ref{tab:table3_2}.  The full methodology can be found in appendix \ref{sec:appendixIII}.

Next, I queried my database to get a distribution for the frequency of HCDR3 lengths. Without removing any redundancies of amino acid sequences, I binned each length and got a distribution, referred to as ``all sequences'' (figure \ref{fig:figure3_7} A). I then removed all redundancies within each donor and plotted them as a function of length, referred to as ``donor unique'' (figure \ref{fig:figure3_7} A, B). Finally, I made a distribution that pooled all the sequences together and removed all HCDR3 redundancies, referred to as ``total unique'' (figure \ref{fig:figure3_7} A-C). The redundancies found in all donors combined, subtracted by all unique sequences, are by shared in at least one donor. For example, for length of HCDR3 equal to 1, I have 174 occurrences when I add up ``donor unique''. That is, for donor 1, I have $X$ amount of unique sequences among that donor added to $X$ amount of sequences for donor 2, etc. However, when I pooled all the sequences of length 1 first, and then removed redundancies, I arrived at 17 ``unique sequences''. That means there are 137 amino acid sequences found in one or more donors. An easier way to view it is to plot the percentage of sequences at a given length that are shared among one or more donor (figure \ref{fig:figure3_7} D).

The mean length of HCDR3 sequences in my dataset was 16.40 $\pm$ 4.03. This finding was in agreement with our previous work using a much smaller dataset \citep{Briney:2012ib}. Although there is no standard definition of a `long HCDR3', I arbitrarily chose the cutoff to be two or three standard deviations from the mean for long or very long HCDR3s, respectively (24AA, 28AA, figure \ref{fig:figure3_7} C). My recombination software (PyIg) also can assign V, D, and J gene segments using the BLAST algorithm \citep{Ye:2013bb}. I observed similar trends in D and J gene family usage as a function of length. D\textsubscript{H}3 and J\textsubscript{H}6 family increased as length of HCDR3 increased (figure \ref{fig:figure3_7} F, G). This finding makes intuitive sense as these are the longest gene segments in their respective families. For V-gene families, I did not observe a difference in germline gene usage as a function of length (figure \ref{fig:figure3_7} E). This trend also was reported with our previous work in which I sequenced the entire antibody segment, however, I can't be absolutely confident in my assignments of V-gene considering I only have on average 20 bp of the V-gene. I can also narrow down individual genes for the D-gene segment and observed an increase in D\textsubscript{H}2-2, D\textsubscript{H}2-15, D\textsubscript{H}3-3, D\textsubscript{H}3-10 and D\textsubscript{H}3-22 as HCDR3 length increased (figure \ref{fig:figure3_7}H).

\begin{figure}
   \centering
   \includegraphics[scale=.5]{images/chapter3/figure3_6.pdf} % requires the graphicx package
   \caption[Overview of HiSeq Scheme]{RT-PCR in round 1 allows addition of an internal index to categorize donors. The cDNA is then subjected to a nested round 2 PCR where the necessary HiSeq plate adaptors and sequencing regions are added that are used by Illumina. The sequencing makes two pair-end reads that are later reconstructed.}
   \label{fig:figure3_6}
\end{figure}

\begin{figure}[!t]
   \centering
   \includegraphics[width=.9\linewidth]{images/chapter3/figure3_7.pdf} % requires the graphicx package
   \caption[Distribution and VDJ Gene Usage]{Length distribution for all productive sequences (black dashed line), unique among within each donor (blue dashed) and unique among every donor (black solid) (A). Zoomed in distribution of ``total unique'' and ``donor unique'' to see overlap. Sequences that are unique between donors are shaded in grey lines (B). ``Total unique'' with standard deviations shown at 20, 24, and 28 (C). Percentage of shared sequence as a function of HCDR3 length. ``Shared sequence'' is defined as being found in two or more donors (D). V-gene families, D-Gene families and J-Gene family frequencies for all sequences, long, and very long HCDR3 sequences (E, F, G respectively). D-gene families can further be divided into individual gene frequencies for all, long, and very long HCDR3 sequences (H).}
   \label{fig:figure3_7}
\end{figure}
\input{tables/chapter3/table3_2.tex}


\section{Addition of Non-Canonical Amino Acids in Rosetta}
The \rosetta software suite was initially developed for \textit{ab initio} folding of small globular proteins using fragment-based search methods and knowledge-based potentials to score models46. Because of this, the code for \rosetta made protein amino acid residues the smallest object. All proteins would be made up of residue objects, and all building blocks would be made up of parameters that describe a residue object. This made sense at the time of \rosetta's inception given its primary purpose, but as its scoring function was being developed for a wide variety of molecular modeling tasks, the residue based code became difficult to implement for non-residue type molecules, \textit{i.e.} drug-type ligands, glycans, and post-translational modifications.

The PG9 complex relied on complex and high-mannose type glycans that were bound to asparagine residues at HIV Env protein position N160 and N156 (HIV strain HXBc2 numbering) \citep{McLellan:2011dg}. Removal of these residues abrogated binding to HIV envelope \citep{Doores:2010gn}. Thus, it was necessary to include both of these glycans in my predictions. It also was necessary to encode these glycans as non-canonical amino acid residues rather than post-translational modification due to the residue object requirement of \rosetta score function. I based the protocol of adding the two non-canonical amino acids after the work of Renfrew and colleagues who developed a generalized protocol for implementing non-canonical amino acids into \rosetta \citep{Renfrew:2012ci}. The protocol used a series of steps that I followed loosely that involved extraction of the residue, minimization of the residue using quantum mechanics simulations, and description of the new amino acid types as a series of parameters that \rosetta~can recognize. I first had to benchmark the non-canonical amino acid types before I could use them in the protocol. To do this, I ran minimization and design against the glycan and determined if my best scoring models were the closest to the native structure found in the PDB. The best scoring model according to total energy was aligned with the native structure. I saw minimal deviation from the native structure indicating that for good scoring models, the glycan conformation is preserved (figure \ref{fig:figure3_8} A). A general trend between the glycan score and the total score as a function of the glycan RMSD (the deviation from the native conformation) was observed, indicating that the glycan deviation follows the \rosetta scoring function (figure \ref{fig:figure3_8} B). I also observed that PG9 HCDR3 amino acids were ideal for antibody-antigen complex during redesign of the PG9-antigen (figure \ref{fig:figure3_8} C).

\begin{figure}[!t]
   \centering
   \includegraphics[width=.9\linewidth]{images/chapter3/figure3_8.pdf} % requires the graphicx package
   \caption[Glycan Addition and Benchmarking Template]{Glycan addition and benchmarking template. Energetically minimized loop of the PG9 HCDR3 overlaid on the native PG9-antigen complex (PDB-3U4E) \citep{McLellan:2011dg}. Native PG9-antigen is shown in darker colors while the redesigned, energetically minimized and re-docked structures are shown in lighter colors. Light chain is shown in blue, heavy chain grey, and antigen in magenta. The two glycans are shown in stick representation. The native glycan positions are shown in transparent stick conformation (A). The score of the glycan and the score of the model are shown as a function of the glycan RMSD. On the y-axis are \rosetta scores, and on the x-axis is the glycan RMSD from native structure found in the PDB. The top panel is the total energy of the complex compared with the glycan RMSD while the bottom compares the glycan score to the glycan RMSD. A general trend is shown between the glycan deviation from native conformation and an increase in score (B). Sequence logo of redesigned HCDR3 loop of PG9 with glycans present. The x-axis shows PG9 native sequence (C).}
   \label{fig:figure3_8}
\end{figure}


\section{High-Throughput Threading of HCDR3 Sequences}
I took 4,000 random HCDR3 sequences from an available dataset of 26,417 (table \ref{tab:table3_2}). These sequences were ``threaded'' over the PG9 HCDR3 backbone and energetically minimized. I did not include the antigen in this first set of modeling experiments as  my first goals were to establish a high-throughput prediction model of PG9-like antibodies and not necessarily anti-HIV gp120 specific neutralizing antibodies. This approach was for an experimental contingency in case my modeling experiments predicted that none of the sequences could bind gp120 antigen.

The threading protocol removes amino acid sequence identity of the HCDR3 loop and replaces, or ``threads'', one of the 4,000 random HCDR3 sequences from my dataset. After the amino acid identity has been replaced, the PG9 antibody is energetically minimized and scored with the \rosetta~scoring function which is detailed in chapter \ref{chap:chapter1}.

Considering each model took approximately 1.2 CPU hours to complete at the computing cluster (https://vanderbilt.accre.com) at the time of writing, all 26,417 models would have taken 31,700 CPU hours to complete. Considering \rosetta takes approximately 1,000-10,000 models to determine energy landscape\citep{Simons:1999wd,Bradley:2005bu,Bradley:2003db,Das:2007em,Raman:2009ht}, the time needed for completion was approaching CPU times from 3.17 x 10$^{7}$  - 3.17 x 10$^{8}$ hours. I could utilize approximately 600 processors running in parallel, cutting down CPU times to 52,834 hours or ~6 years. Obviously these numbers were unrealistic at the time of writing, so I decided to optimize heuristics methods that could be accomplished with a far fewer number of models. I chose 4,000 random sequences with 50 models as a strict cutoff to maximize data output within the capabilities of my computational resources.

To evaluate the threading protocol, I scored my models using the \rosetta scoring function and plotted it against the deviation from PG9's native structure (figure \ref{fig:figure3_9} A). As a control, PG9 and PG16 also were put into the threading protocol to evaluate the scoring function's ability to make distinctions between poor scoring and favorable scoring models. Considering I know that PG9 and PG16 sequences form PG9's backbone conformation, I expected a very low deviation from PG9's crystal structure conformation. I also expected \rosetta's scoring function to score these sequences as favorable relative to many of the random HCDR3s that were evaluated in the protocol. Indeed, I observed that PG9 and PG16 consistently ranked as the most favorable in terms of score and conformation and confirmed the accuracy precision of my threading protocol (figure \ref{fig:figure3_9} A).

In contrast, the randomly selected 4,000 HIV-\naive HCDR3 sequences tended to group into three separate categories: 1) Those sequences which maintain the HCDR3 structure of PG9 but scored poorly (figure \ref{fig:figure3_9} B), 2) sequences that scored well but collapsed or deviated away from PG9's native conformation (figure \ref{fig:figure3_9} D), and finally 3) sequences that scored well and retained PG9's native conformation (figure \ref{fig:figure3_9} C). These three groups gave a diverse sequence-energy landscape to be carried on into my heuristics in order to evaluate the remaining ~22,000 HCDR3 sequences.

\begin{figure}
   \centering
   \includegraphics[width=.9\linewidth]{images/chapter3/figure3_9.pdf} % requires the graphicx package
   \caption[Threading PG9 Produces Three Structural Outcomes]{The threading protocol for PG9 was evaluated for structural mimicry using the \rosetta scoring function. The models scores were evaluated and plotted against the root mean squared deviation (RMSD) from the native PG9 HCDR3 structure. Lower scoring models are closer to native structure (y-axis) while models closer to PG9 HCDR3 structure have a lower RMSD (x-axis). Repeated measures of PG9 and PG16 were grouped close to native structure and close to PG9 structure (blue and red points) while HIV-\naive donor antibody sequences are shown are labeled HD (black points) (A). For figures B-D, HCDR3 structures were aligned to native PG9 (shown in transparency). The HCDR3s are colored by their \rosetta score with blue being a favorable scoring residue and red being an unfavorable score. There were three different outcomes observed for the threading experiment. Sequences that retain PG9's structure but produce an unfavorable score (B, dashed red circle in figure A), sequences which produced a favorable score but collapsed away from PG9 native structure (D, dashed orange circle in figure A), and sequences that scored favorably and adopted PG9 conformation (C, blue dashed circle in figure A).}
   \label{fig:figure3_9}
\end{figure}


\section{Heuristics to Rapidly Score HCDR3 Sequences}
Using the selected 4,000 HIV-\naive HCDR3 sequences, I randomly chose half of the sequences to be in the benchmark set or the training set. For the training set, 2,000 sequences were evaluated with the \rosetta~scoring function. For each amino acid position 96-125 (PDB numbering), I filled a position structure specific scoring matrix (P3SM). For each position, and each amino acid identity, I gave an initial score of 0.0 to baseline yielding a 20 x 30 matrix, and averaged each amino acid identity score at each position (figure \ref{fig:figure3_10} A,B). The matrix can be visualized as a heat map with favorable scores in blue and unfavorable scores in red (figure \ref{fig:figure3_10} B).

The P3SM became my heuristic to rapidly score other sequences that were not run through the computationally expensive threading protocol. I could validate my heuristic by evaluating how well the P3SM predicted \rosetta~energies by applying it to the other 2,000 sequences in the benchmark set. The P3SM may not give absolute \rosetta energies but should provide a relative ranking compared to other HCDR3 sequences. I observed a 0.863 r$^{2}$-value for a correlation between actual \rosetta~energies and P3SM predicted energies. For the top 10\% energies evaluated by the \rosetta~energy function, my correlation fell to 0.300 (figure \ref{fig:figure3_10}C).

To determine the relative noise of the P3SM, I ranked the scores and determined the position of PG9. It was found in 104th position out of all 26,417 sequences (0.3\%). The difference in the PG9 P3SM score and the top scoring sequence was 3.82 \rosetta~energy units (REU). Thus, the final noise tolerance of my matrix was -34.84 $\pm$ 3.82 (-31.02 to 38.66) that encompassed approximately 1000 sequences (figure \ref{fig:figure3_10} D). I then defined my P3SM heuristic to be able to accurately pick out the top 1000 out of 26,417 sequences (3.79\%).

\begin{figure}[!t]
   \centering
   \includegraphics[width=.9\linewidth]{images/chapter3/figure3_10.pdf} % requires the graphicx package
   \caption[Heuristics Predict HCDR3 Sequences that Mimic PG9 Structure]{Heuristics predict HCDR3 Sequences that mimic PG9 structure. 2,000 random models were evaluated at each amino acid position. A cartoon representation of the HCDR3 loop is shown with each position colored with a blue-red gradient according to favorable to unfavorable amino acid identities, respectively (A). Each position was initially assigned a score of 0.0 and enumerated through averaging each amino acid identity with the score to fill a position structure specific scoring matrix (P3SM) from 2,000 HCDR3 sequences. The matrix was then used to rapidly score the other 2,000 models to predict a \rosetta~score (B). A simple linear regression was applied to the actual \rosetta~energy score of a sequence and it's predicted score to give a coefficient of determination (r$^{2}$) of 0.863. When only the top 10\% by actual \rosetta score were considered the coefficient of determination dropped to 0.330 (C). Using the P3SM, PG9 scored 104th. Calculating the difference in score between the top scoring sequence and PG9 left a noise tolerance of 3.82 \rosetta energy units (REU). Calculating $\pm$ 3.82 REU of PG9 left 1,000 HCDR3 sequences that fell within the noise tolerance range of the P3SM (D).}
   \label{fig:figure3_10}
\end{figure}


\section{Docking and Minimization of HCDR3 Variants}
With 1,000 candidate sequences from the original sequence pool of 26,417, I ran the threading protocol again in the presence of the antigen and complex glycans that were added from earlier sections (see addition of non-canonical amino acids). The threading protocol was modified slightly to include all atom constraints, and an additional docking step and 100 models generated for each sequence (100,000 models). The full protocol is detailed in the Appendix section \ref{sec:appendixIII} with a protocol capture in section \ref{sec:appenixV}. In addition to PG9 deviations and total scores, I also evaluated binding energies, which are the change in free energy ($\Delta\Delta$G). I defined the $\Delta\Delta$G as follows:

\begin{gather*}
    \Delta \Delta \textup{G} = \Delta \textup{G}_{\textup{Bound}} - \Delta \textup{G}_{\textup{Unbound}} \\
    \textup{Where,} \\
    \Delta \textup{G}_{\textup{Bound}} = \textup{RosettaScore}_{\textup{Complex}} \\
    \textup{and}\\
    \Delta \textup{G}_{\textup{Unbound}} = \textup{RosettaScore}_{\textup{Separated}}
\end{gather*}

I decomposed the binding energies, total energies, and deviations into several metrics to evaluate the models. Total energy, binding energy, and deviation (C$\alpha$ - RMSD) for just the HCDR3 portion of the model, the binding energy contribution by both of the glycans (N156 $\Delta\Delta$G and N160$\Delta\Delta$G, HIV strain HxBC2 numbering), and the total binding energy for the entire model. Initially each of these metrics was evaluated individually to see where they ranked or in pairs by plotting them against each other (figure \ref{fig:figure3_11}, left panel). Favorable sequences were near PG9 in the plot. I also plotted HCDR3 metrics as heat maps and score models qualitatively (figure \ref{fig:figure3_11}, right panel). Both of these methods were inefficient, as each metric produces a different rank ordering of the HCDR3 sequences. Therefore, I sought to combine these six metrics into one score to easily compare where each sequence ranked in comparison to PG9. To combine the metrics, I assigned a simple Z-score to each metric to find out where that model ranked. The Z-score was weighted and averaged for each scoring metric to produce a weighted Z-score that was used to efficiently rank sequences with one score using the following equation:


\begin{gather*}
\textup{Weighted-Z} = \frac{\sum_{i}^{N}w_{i}\times Z_{i}}{N}
\end{gather*}

\noindent where w\textsubscript{i} is the weight of each Z-score statistic i, Z\textsubscript{i} is the z score for the statistic i, and N is the total number of statistics. The addition of the weights allowed optimization during \textid{ad hoc} analysis to ensure PG9 and PG16 were the most favorable weighted Z-score. The final weights used in the protocol were:
\begin{enumerate}
\setlength{\itemsep}{0pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
\item Total $\Delta\Delta$G \textemdash{}  $3.0$
\item HCDR3 $\Delta\Delta$G \textemdash{}  $1.0$
\item HCDR3 Total Score \textemdash{}  $1.0$
\item HCDR3 $\textup{C}_{\alpha} \textup{RMSD}$ \textemdash{}  $0.5$
\item $\textup{N156} \Delta \Delta \textup{G}$ \textemdash{}  $0.5$
\item $\textup{N160} \Delta \Delta \textup{G}$ \textemdash{}  $0.5$
\end{enumerate}

\noident I chose the top 80 HCDR3 HIV-\naive donor antibody sequences to carry on to experimental steps, as this number was my upper limit to synthesize.

\begin{figure}
   \centering
   \includegraphics[width=.9\linewidth]{images/chapter3/figure3_11.pdf} % requires the graphicx package
   \caption[Scatter Plots and Heat Maps for P3SM Threading Analysis]{Scatter plots and heat maps for P3SM threading analysis. Each metric was plotted on a separate axis and sequences which were found to score well were found in the lower left of the graph. For example, HCDR3 binding energy is plotted against HCDR3 score. PG9 and PG16 were found to have favorable binding energy and score (left panel). A heat map was generated for each metric for residues in the HCDR3. For example, contribution to binding energy for each amino acid identity was plotted as a heat map. The red-blue scale is for neutral to favorable reactions (right panel).}
   \label{fig:figure3_11}
\end{figure}

\section{Clustering of HIV-Na\"{i}ve  Sequences}
Rather than synthesize all 80 HIV-\naive donor antibody sequences that were predicted to be closest in structure to PG9, I instead considered that several of the sequences were related siblings to each other. Indeed, upon clustering at a threshold 85\% amino acid similarity (4 mutations), the sequences clustered into nine different groups, while five sequences formed independent groups (figure \ref{fig:figure3_12}).

I next aligned the nucleotide and amino acid sequences to find sequence similarities among antibodies in each of my PG9 clustering groups. Surprisingly, only the beginning and end of the sequences corresponding to the base of the HCDR3 displayed a high degree of similarity (figure \ref{fig:figure3_12} A-B).The similarity arises from the in-frame J\textsubscript{H}-6 gene for most long HCDR3 sequences, with the exception of cluster I and IG2, which used J\textsubscript{H}-4 and had evidence for significant J-gene exonuclease activity, respectively (table \ref{tab:table3_3}). I also detected some nucleotide conservation for positions 27-36 corresponding to amino acid 9-12, which resulted in a semi-conserved SSGY motif (figure \ref{fig:figure3_12} A-B). Rather than synthesize and express each of the 80 variants, I chose to synthesize one member from each cluster and one of the sequences that did not cluster. These sequences were selected based on their \rosetta weighted Z-score metric within each cluster (table \ref{tab:table3_4}).

\begin{figure}[!t]
   \centering
   \includegraphics[width=.99\linewidth]{images/chapter3/figure3_12.pdf} % requires the graphicx package
   \caption[PG9-Mimicry Candidates Cluster Into Groups]{PG9-mimicry candidates cluster into groups. The consensus nucleotide sequence were aligned for each cluster B-J. There was little sequence similarity in the junctions except for the V\textsubscript{H} and J\textsubscript{H}  gene. Sequence logo representations are shown above the sequence to detect conservation. Five independent group sequences are also shown (A). The same as (A) translated. Conserved elements are shown in the J\textsubscript{H} region due to conserved use of the J\textsubscript{H}6 gene family (B). The cladogram for each HCDR3 amino acid sequence shows how the top 80 sequences clustered into 9 groups with some groups having multiple sequences that differ by 1-4 amino acid mutations but were derived from the same lineage (C).}
   \label{fig:figure3_12}
\end{figure}
\input{tables/chapter3/table3_3.tex}

\section{Design of Top PG9-Mimicry Candidates}
I realized that the models for these wild-type sequences sometimes contained clashes or caused other strain in the HCDR3 loop that was reflected as a difference in normalized Z-scores (table \ref{tab:table3_4}) or as an energetic gap between the predicted energies of PG9 and the top 80 variants selected (figure \ref{fig:figure3_13} A). Because of this observation, I decided to run a dock-design protocol that would relieve clashes and strains for unfavorable amino acids. I also imposed a filter that gave bonuses for amino acids that were native to the starting sequence. In this way, I was able achieve a higher level of PG9 mimicry by lowering the energetic gap while retaining as many wild-type amino acids as possible (figure \ref{fig:figure3_13}). I chose a variant from each cluster whose sequence recovery (the percentage of designed sequences that were wild-type) and carefully analyzed each predicted mutation suggested by \rosetta. I ranked the mutations based on a fitness score. I defined the fitness core as the change in total score for the mutation from the wild-type added to the change in binding energy for the mutation compared to wild-type. If the fitness was found to be significant, the mutation was confirmed by visual inspection in PyMoL (\textit{i.e.} if \rosetta~predicts more hydrogen bonds, are there more hydrogen bonds).

Briefly, I will explain the rationale for choosing the mutations for cluster B. Variant VU42252 was chosen as the representative candidate for design since it provided the best sequence recovery to the wild-type sequence as well as beneficial fitness (figure \ref{fig:figure3_14} A). Each mutation is plotted as a function of its fitness and grouped together by position (figure \ref{fig:figure3_14} B). If there was a large decrease in energy from the wild-type sequence, it corresponded to an increase in fitness for a given mutation. Those mutations that had a favorable change in fitness with a magnitude of greater than 1.5 \rosetta energy units were manually inspected with PyMoL (figure \ref{fig:figure3_14} C,D). Two such mutations are shown for position 106 (figure \ref{fig:figure3_14} C, PDB numbering) and position 120 (figure \ref{fig:figure3_14} D, PDB numbering). For position 106, the wild-type serine was not preferred since it leaves a large exposed gap between the antigen face and the antibody interface. Upon mutation to an asparagine, the gap was filled in. Additionally the asparagine makes new hydrogen bond contacts with the antigen. This finding was predicted to benefit the binding energy and stabilization of the complex. Position 120 has a wild-type tyrosine that repels against the large steric bulk of the glycan. A mutated asparagine at this position allows an inter-HCDR3 hydrogen bonds to stabilize the loop while also making additional hydrogen bonds to the glycan.

For cluster B, I chose six different positions to be mutated alone or in combination that produced five different candidate variants. This proposed a panel of six different variants to validate for cluster B, one wild-type sequence, and five mutational variants that ranged from 3-10 mutations. The remaining clusters were analyzed similarly and in total, 10 wild-type sequences were chosen for experimental characterization, and 74 different sequences that were mutational variants of the representative cluster sequence were chosen.

\input{tables/chapter3/table3_4.tex}
\begin{figure}[!t]
   \centering
   \includegraphics[width=.9\linewidth]{images/chapter3/figure3_13.pdf} % requires the graphicx package
   \caption[Energetic Barriers for Complete PG9-Mimicry]{Energetic barriers for complete PG9-mimicry. The total HCDR3 energy and binding energy are plotted for the top 80 sequences selected by normalized Z-score as well as PG9 and PG16. There was a significant energetic gap between these sequences and complete PG9 mimicry (A). After redesign, the sequences approached the energy of PG9 and PG16 (shown as connecting triangles to circles). The blue-red scale is the sequence recovery percentage, indicating how much of the wild-type sequence was retained.}
   \label{fig:figure3_13}
\end{figure}

\begin{SCfigure}
   \centering
   \includegraphics[scale=.8]{images/chapter3/figure3_14.pdf} % requires the graphicx package
   \caption[Mutation Analysis of Cluster B]{Mutation analysis of cluster B. A sequence logo representation of the cluster B variant with the best sequence recovery is shown. The wild-type sequence of the cluster B candidate named VU42252 is plotted on the x-axis. The preferred mutations for each position in the HCDR3 are shown on the y-axis with the height of each letter corresponding to \rosetta`s preference for that mutation (A). Each mutation that was predicted to benefit fitness is plotted by position. The more preferred mutations correspond to a more negative number. A cutoff threshold of 1.5 \rosetta energy units is shown as a dashed line to indicate mutations that were considered for experimental characterization after manual inspection (B). Manual inspection of the mutation at position 106 from the wild-type serine (left panel) to asparagine (right panel). The antigen is shown in beige or gray, for the wild-type or mutation, respectively. The surrounding residues are colored in yellow. The HCDR3 loop is colored blue. The asparagine hydrogen bonds to the antigen, which favors binding energy and stabilization of the HCDR3 loop (C). Manual inspection of position 120 is plotted with the HCDR3 loop in light blue, the heavy chain in green, and the light chain in cyan. The glycans are shown in stick representation. The designed asparagine compared to the wild-type tyrosine makes inter-HCDR3 stabilizing hydrogen bonds as well as additional bonds with the glycan (C).}
   \label{fig:figure3_14}
\end{SCfigure}

\section{Synthesis and Screening of PG9-Mimics}
For the 84 variants, I chose a synthesis strategy that would allow us to swap in HCDR3 sequences with unique cloning sites rather than resynthesize the framework and HCDR1-2 sequences redundantly. It was to this end that I introduced two unique cloning sites at the 5' end and the 3' end of the HCDR3 sequence. Using these cloning sites, small gene fragments could be synthesized rapidly and cost-effectively.  I did not expect that all of the 84 variants would express or bind HIV gp120 and decided to screen each candidate for binding and expression on a small scale.  I made 30 mL test cultures for recombinant antibody expression to screen the supernatant for the presence of IgG antibody and its ability to bind gp120 protein. According to the literature cited by McLellan \texit{et al.}, only certain gp120 monomers bind to PG9 using enzyme-linked immunosorbent assays (ELISA), while PG16 binds gp120 monomers very weakly  \citep{McLellan:2011dg}. Therefore, I combined fifteen different gp120 protein variants into an antigen cocktail to maximize the chances of binding to each of my 84 PG9-mimics. This cross screening for antibody expression and antigen binding was designed to pre-emptively select which of my 84 candidates should be carried on to upscaled expression.

Figure \ref{fig:figure3_15} plots my screening metrics for expression and binding. I binned each candidate to either be further characterized, or not. I based this decision on an expression cutoff of at least 300 $\mu$g/L and a binding of at least 1 OD in ELISA at maximal concentration.  For some antibodies that did not express well but showed some binding activity, I chose to carry on to further experimentation. The results are summarized in Table \ref{tab:table3_5} with 35 variants to be expressed at a larger scale. For these, I choose either a 300 mL expression (10-fold expansion of volume) or a 1L (33-fold expansion of volume) expression based on initial expression levels. These antibodies were purified and carried on to biophysical characterization and neutralization studies. With the exception of cluster I, there was at least one candidate from every cluster that would be further characterized (table \ref{tab:table3_5}).

\input{tables/chapter3/table3_5.tex}
\begin{figure}[H]
   \centering
   \includegraphics[width=.9\linewidth]{images/chapter3/figure3_15.pdf} % requires the graphicx package
   \caption[Expression and Binding of 84 Variants]{Expression and binding of 84 variants. Supernatants were tested for binding for all 84 variants and plotted on the y-axis. The x-axis is the level of expression for each variant. Plus symbols denote wild-type sequence, squares are mutants. I color by cluster (NC is the non-clustered or independent group). Only variants that bound tightly and expressed well were considered for upscaled expression and further characterization.}
   \label{fig:figure3_15}
\end{figure}

\section{Biophysical Characterization of PG9-Mimics}
I chose to use 8 gp120 monomers based on the ability of PG9 to bind them using my ELISA experimental conditions. These monomers contained 5 clade B variants (BaL.01, SC422661.8, 6535.3, RHPA4259.7, and TRJO4551.58), 2 clade C variants (CAP45.2.00.G3 and ZM109F.PB) and 1 laboratory adapted SHIV chimera (HXBc2P3.2). I found that PG9 bound these 8 candidate monomers based on previous literature \citep{McLellan:2011dg} and my own pilot studies (chapter \ref{chap:chapter4}). I coated each plate in a 384-well format with gp120 monomer and tested binding to the purified 35 variants I analyzed based on my expression/binding sieve analysis in the previous section. Due to replication of these experiments, and consequently the amount of protein used, 4 variants were lost leaving, a total of 31 variants analyzed. I calculated effective concentration at half-maximal binding (EC\textsubscript{50}) of each variant against 8 gp120 monomers. I started my dilution at a very high concentration in order to detect weakly binding antibodies. I set my cutoff for binding at EC\textsubscript{50} less than 100 \mcml.

Not surprisingly, most variants did not bind with the same breadth and potency of PG9. In fact, 16 of the variants characterized did not reach my threshold of binding for any of the monomers tested (figures \ref{fig:figure3_16}, \ref{fig:figure3_17}). However, I found at least one gp120 monomer that bound fifteen of my variants with an \ec less than 100 \mcml. I find the easiest monomers to bind were BaL.01, ZM109F.PB, and CAP45.2.00.G3. My broadest antibody was VU43171\_6MUT from cluster C, which bound 7 out of 8 gp120   monomers (figures \ref{fig:figure3_16}, \ref{fig:figure3_17}). My most avid antibody was VU28693\_5MUT from cluster E, which bound ZM109 and BaL.01 at a concentration of 1 \mcml. For comparison, PG9 binds ZM109 or BaL.01 with an \ec of 0.03 \mcml or 0.06 \mcml, respectively. I also found a completely wild-type antibody, that is, an antibody with no design modifications from \rosetta, which bound with an \ec to BaL.01 or ZM109 at 3.7 \mcml or 3.6 \mcml, respectively.

\begin{sidewaysfigure}
   \centering
   \includegraphics[scale=.7]{images/chapter3/figure3_16.pdf} % requires the graphicx package
   \caption[Expressed Candidate ELISA Binding]{Expressed candidate ELISA binding. The y-axis is the absorbance for each ELISA binding curve, x-axis is the log of concentration. Each panel represents a different antibody out of the 31 variants tested. Different monomers are represented by symbols. PG9 wild-type (PG9\textit{wt}) is shown at the bottom right as a reference.}
   \label{fig:figure3_16}
\end{sidewaysfigure}

\begin{figure}[!ht]
   \centering
   \includegraphics[width=.99\linewidth]{images/chapter3/figure3_17.pdf} % requires the graphicx package
   \caption[Heat-Map of ELISA Binding to Gp120 Monomers]{Heat-map of ELISA binding to gp120 monomers. Each cell represents the effective concentration at half-maximal binding for each variant bound to 8 representative viruses. The red-white-blue scale is a gradient from the maximum \ec, to the minimal, 95 \mcml to 0.03 \mcml, respectively. I show the ID, a unique identifier, the cluster each antibody falls in, and the name of the antibody that indicates how many mutations from the wild-type sequence were present. The antibodies are sorted by their strength of binding to BaL.01.}
   \label{fig:figure3_17}
\end{figure}

\section{Neutralization of HIV by HIV-Na\"{i}ve  PG9-Mimics}
From the literature, I know that changes in the HCDR3 loop sequence can dramatically alter specificity \citep{Pancera:2010hh,Pejchal:2010fp,Pancera:2013ev}. For instance, making a PG9 HCDR3 with a PG16 backbone was able to neutralize additional viruses, while PG16 HCDR3 with PG9 backbone also was able to pick up additional breadth \citep{Pejchal:2010fp}.
The exact molecular mechanisms that allow PG9 to bind most gp120 monomers and PG16 weakly bind gp120 monomers are currently unknown\citep{McLellan:2011dg}. Both PG9 and PG16 also do not bind gp120 monomers there are able to neutralize indicating their preference for a trimer specific conformation. It was because of this, I decided to test 31 out of my 35 HCDR3 variants for neutralization against pseudoviruses which present native trimer \citep{Montefiori:2005jt}. I decided to send my variants to a collaborator, Dr. David Montefiori, who designed and performed multiple neutralization assays to test recombinant HIV variants \citep{DeCamp:2014jd}.

It would stand to reason that the relatively large EC\textsubscript{50} returned in the binding studies would generate equally large \ic~from the neutralization assays. It was because of this, that I decided to start the concentrations at 100 \mcml~for the neutralization screen. This made my neutralization screen protein-limited and at the time of publication, I could only test two viruses, RHPA and RHPA.N160A. From figure \ref{fig:figure3_17}, it is evident that this virus is difficult to neutralize and are currently pursuing studies of much weaker viral strains derived from the literature and figure \ref{fig:figure3_17}. Regardless, VU30400\_7MUT from group G was able to neutralize RHPA at 49.3 $\mu$g/mL while PG9 neutralized at 6.22 $\mu$g/mL. Both of mAbs lost the ability to neutralize RHPA.N160A, a knockout mutation for this class of antibody which removes the large glycan at position 160 indicating its mimicry of PG947.

\section{Analysis of Mutations with Rosetta}
Lastly, I wanted to analyze the binding antibodies for sequence conservation to see if there were any trends. Indeed, figure \ref{fig:figure3_18} shows antibodies that bound BaL.01 with an EC\textsubscript{50} less than 55 $\mu$g/mLL. The neck of the HCDR3 loop showed great sequence conservation for residues 95-97 with a VRE or VRD conserved sequence, and residues 118-125 with a YYYYMDV, the wild type sequence of an unmutated J\textsubscript{H}6 gene. These trends were observed for long HCDR3 class of antibodies in recent clustering studies \citep{North:2011dv}. I was more interested in conserved sequence elements that would be at the antigen interface. For those residues, there was great sequence divergence and only a few conserved elements were observed. An aromatic residue at position 104, an asparagine, glycine, and tyrosine and for positions 106-108. These sequences fall at a critical hairpin loop that is necessary to make the crucial turn in order to align the beta-sheets necessary for making contact with the antigen. Indeed, \rosetta preferred very few residues at this position due to the non-canonical torsions that were adopted. Other than those sequence elements, I see great sequence diversity, particularly in the contact residues predicted from the crystal structure and homology models (figure \ref{fig:figure3_18}).

\begin{figure}[!t]
   \centering
   \includegraphics[width=.99\linewidth]{images/chapter3/figure3_18.pdf} % requires the graphicx package
   \caption[Mutational Analysis of HIV-\naive Binding]{Mutational analysis of HIV-\naive binding. HCDR3 sequences are shown for variants that had \ec values less than 55\mcml~and ranked 1-13 according to their potency. PG9 is the most potent and serves as a reference. Conservation is shown as an identity plot above the sequence alignment. Sequences are colored white-black according to their consensus conservation. The asterisks below the sequence alignment show contact residues according to the native crystal structure. Representative structures of conserved elements including an aromatic at position 104, an asparagine at position 106, and a tyrosine at position 108 are shown with the antigen in black, the HCDR3 in blue, and the interacting residues shown in yellow stick conformation. The aromatic residue buries a hydrophobic residue against the loop creating $\Pi-\Pi$ stacking against the nitrogen backbone. N106 is preferred at the torsion necessary to adopt the loop $/Beta$-turn. A tyrosine at position 108 hydrogen bonds with a glutamate on the antigen face stabilizing the structure.}
   \label{fig:figure3_18}
\end{figure}

\section{Conclusions and Future Directions}
A protective vaccine against HIV-1 will likely involve an elicitation of a broadly neutralizing serum \citep{Mascola:1999wk,Burton:2012bh,Hessell:2009ig,Hessell:2010kq,Hessell:2009fy,Hessell:2012kq}58-63. There are a limited number of neutralizing targets for these bNAbs and include the CD4-binding site, the V3-loop, the V1/V20-loop, membrane proximal region (MPER), and the outer N332 glycans \citep{Walker:2010bm}. Here, I aim to target the V1/V2 loop due to the ubiquity of patients infected with HIV to target this region with neutralizing antibodies \citep{Walker:2010bm,Gray:2009jw,Lynch:2011ea,Georgiev:2013kd}. As discussed in the introduction, the RV144 trial, the only HIV vaccine trial to date to show modest efficacy, showed that the correlates of protection were the elicitation of V1/V2 binding mAbs and selective pressure on the V2 region of HIV env \citep{Rolland:2012iz,Haynes:2012ka}.

Recently, a long HCDR3 V1/V2 binding pathway has been elucidated for potent bNabs \citep{DoriaRose:2014ic}. A patient from the CAPRISA cohort was studied longitudinally since initial HIV infection and the researchers found a modestly potent neutralizing antibody at week 58 post-infection with an HCDR3 length of 35 amino acids. In parallel the researchers had also been taking PBMC samples at various time points throughout infection to find the pathway for the development of these neutralizing antibodies. Using pyrosequencing, they found an unmutated antibody at week 30-38 with no VH or VL gene mutations. Longitudinal sequencing analysis shows this antibody mutates away from the unmutated common ancestor (UCA) and picks up potency with a total of 14 mutations in the HCDR3 regions (40\% mutation) at 58 weeks, but relatively is  relatively germline in the other regions (16\%). The researchers showed 54\% mutations from the UCA in the HCDR3 with a neutralization breadth up to 47\%. This study shows the tremendous range of sequence diversity that can converge onto one epitope while maximizing breadth and potency. This study is exceptional in its design but does leave some unanswered questions. Firstly, they only derive their antibodies from V\textsubscript{H}3-30 sequences. This leaves out a tremendous amount of potential recombinations that could become neutralizers. It also focuses on one UCA phylogeny and patient. Although the UCA is said to be available at the original time of recombination, there is no evidence that it is in completely HIV-\naive individuals as it was detected at 30-38 weeks post-infection. This is not a true HIV-\naive study as it attempts to characterize the developmental pathways of V1/V2 binders post-infection.

 I interrogate the long HCDR3 repertoire prior to infection rather than post-infection. I use 64 different donors maximizing my sequence pool and diversity. I also use a deeper sequencing method to get the depth necessary to characterize such a broad repertoire. My elegant combination of computational design with bioinformatically-driven heuristics allows me to interrogate the tremendous sequence diversity of the HIV-\naive repertoire. I aimed to answer two simple questions: do HIV-\naive donors possess long HCDR3 sequences that potentially bind and neutralize HIV? If not, will minimal mutations allow them to bind and neutralize V1/V2 epitopes?

The approach to answering these questions involved a four-part strategy that marries computational and experimental methods in order to investigate the HIV-\naive repertoire. First, I used deep sequencing to accumulate a vast pool of HCDR3 sequences. I then used bioinformatics analysis with new algorithms to determine 30-length HCDR3 sequences. Using \rosetta, I used these sequences to establish a heuristic that would let us rapidly evaluate 30 length HCDR3 sequences for their ability to form a PG9-type loop. This allowed us to trim down my vast sequence pool to a manageable number of HCDR3 sequences to study experimentally. \rosettadesign allowed me to simulate the process of somatic mutation by applying minimal designs to my sequences in order to enhance potency and breadth.

I experimentally characterized 84 variants, a combination of 10 clusters returned from the computational predictions and 74 combinations of mutations predicted to enhance binding. Of those, I trimmed the number down to 31 due to a lack of expression or initial binding. Of the 31 antibodies, I performed ELISA experiments on 8 representative monomers finding that a total of thirteen including two wild-type sequences have an \ec less than 50 \mcml, well within my tolerance to be considered a binder. For my neutralization work, one antibody neutralized a tier-2 virus with a 7-fold lower potency than PG9. I expect that many more of my variants will neutralize under optimal experimental conditions. The tier-2 virus is ambitious for my studies of HIV-\naive binders and I expect that a tier-1 virus, may be potentially easier to neutralize.

My work has several implications for vaccine design as it demonstrates that multiple HIV-\naive donors contain long HCD3s with the ability to bind gp120. This demonstrates how close an HIV-\naive donor is to actually mimicking a mAb that is known to be a broad and potent V1/V2 binder. It was long thought that a repeated vaccination schedule that would gradually induce the necessary somatic mutations to recapitulate the broad and potent antibodies that bind the CD4-binding site. Here I show that an average HIV-\naive donor is closer to producing a broadly neutralizing mAb than initially hypothesized \citep{Mikell:2011hr}. This potential paradigm shift in vaccine design would aim to prime for these B-cells with long HCDR3s and then boost for specificity offering protection from HIV-1 challenge.

