\section{Appendix III - Materials and Methods}
\label{sec:appendixIII}
\par\vspace{10pt}
\subsection{Chapter \ref{chap:chapter2} - Materials and Methods}
\par\vspace{10pt}
\subsubsection{Selection of Antigen-Antibody Complexes}
Diverse antigen-antibody complexes were collected from the Protein Data Bank (PDB; \url{www.pdb.org}) in which antibodies in different complexes were derived from the same predicted heavy chain variable gene segment. Candidate complexes were queried from the protein databank using the IMGT-3D structural query editor for immune system receptors \citep{Kaas:2004kv}. PDB structures were used as design candidates if they met the following criteria: 1) the antibody was encoded by a V\textsubscript{H}1-69, V\textsubscript{H}3-23, or V\textsubscript{H}5-51 gene segment, 2) the structure contained a human immunoglobulin, and 3) the ligand type was a protein complex. The search yielded 10, 8, or 3 antibody-antigen complexes encoded by the heavy chain variable gene segments V\textsubscript{H}1-69, V\textsubscript{H}3-23, or V\textsubscript{H}5-51, respectively. Nature of the antigen and antibody isotype were not considered in the selection as the 21 complexes represent an exhaustive search of the PDB for these gene-segments. The gene segments were aligned using the ClustalW2 multiple sequence alignment algorithm \citep{Larkin:2007hz}. Each input structure was energetically minimized using the \rosetta scoring function but constrained to PDB input backbone coordinates \citep{Das:2007em}.

\subsubsection{Multi-state Design of Antigen-Antibody Complexes}
Three design experiments were performed, one for each of the three germline segments (V\textsubscript{H}1-69, V\textsubscript{H}3-23, or V\textsubscript{H}5-51) using the multi-state design mode of the \rosetta algorithm and scoring functions. I adapted a generalized multi-state design protocol that was described in detail previously that perform design on multiple antibody-antigen complexes at once \citep{LeaverFay:2011ji}. Briefly, each computational design experiment computed an optimal sequence predicted to define a low-energy structure.  In the multi-state design experiments, an energetic consensus sequence for all of the states was predicted, rather than treating each state as a separate entity. The energy for a given sequence was computed and designated the ``design fitness'' for all states. The corresponding amino acids were derived from the alignment (\textit{e.g.}, heavy chain amino acid 5 on complex A corresponded to heavy chain amino acid 5 on complex B). The details of the multi-state algorithm is described elsewhere \citep{LeaverFay:2011ji}.

\subsubsection{Single-State Design of Antigen-Antibody Complexes}
Single-state design was performed using the \rosetta~multi-state application. The algorithm was altered so that only one complex was considered for each of the 10, 8, or 3 design experiments with V\textsubscript{H}1-69, V\textsubscript{H}3-23, or V\textsubscript{H}5-51 complexes, respectively.

\subsubsection{Design Analysis of Multiple- or Single-State Design}
For each design experiment, 100 independent design trajectories were calculated. Sequence logos then were generated using the Berkley web-logo server (\url{http://weblogo.berkeley.edu/})~\citep{Crooks:2004do}. Information for each sequence logo can be extrapolated as follows extending the work of Schneider \textit{et al.} \citep{Schneider:1990ub}. For each variable position, the probability of seeing each of the 20 naturally encoded amino acids p\textsubscript{i} was computed and compared with the background probability p\textsubscript{b} = 1/20 = 5\%. To quantify the deviation of the observed probability from the background probability I compute the self-information for each of the 20 amino acids as I\textsubscript{i} = p\textsubscript{i} x log\textsubscript{2}(20 x p\textsubscript{i}) in `bit'. If the amino acid occurs as often as expected from the background probability, I\textsubscript{i} is zero. Ii becomes larger if the amino acid is over-represented and approaches 4.32 if p\textsubscript{i} = 100\%. A total bit-score for the sequence design was obtained by summing all individual bit-scores for each amino acid. The bit-scores for the target sequence then were analyzed, and statistics were computed using Prism software version 5.0 (GraphPad Software).  For comparisons between germline sequence and mature sequence within the same design experiment, a Wilcoxon matched pairs test (non-normal, paired t-test) was used to compute the p-value at 99\% confidence level. For comparison between design experiments, a student's paired t-test was used to compute the p-value at 99\% confidence level.

\subsubsection{Amino Acid Environment}
The neighbor vector algorithm quantitatively determines the surface-exposure of a given residue and is described by Durham and colleagues elsewhere \citep{Durham:2009kt}. Briefly, each C$_{\beta}$ is computed to a vector and each vector is given a score based on the number and orientation of each C$_{\beta}$in the proximity. The weight of each neighbor falls of as a function of distance.
For interface scores, the change in neighbor vector was used, where the neighbor vector score of the amino acids in the unbound antibody is subtracted from the neighbor vector scores of the complex. Interface residues would have a large change in neighbors and proportional to the change in neighbor vector score.

\subsubsection{Phi-psi Angle Calculations}
All V\textsubscript{H} framework residues were grouped by complex. For each residue, phi-psi angles and secondary structure classification were determined using DSSP \citep{Kabsch:1983bp}. For each residue position across all complexes considered in design, the standard deviation of the phi-psi angles was calculated if they were included in the beta-sheet framework. A student's t-test was performed between the standard deviations between residue positions that recovered to germline (bit-score > 1), or did not recover to germline (bit-score < 1). For a reference, a deviation for all framework beta-sheet positions was also calculated for all residues even if they were not included in the design protocol.
\clearpage
\subsection{Chapter \ref{chap:chapter3} - Materials and Methods}
\par\vspace{10pt}
\subsubsection{RNA Extraction}
Peripheral blood mononuclear cells were isolated from 64 HIV-uninfected individuals (HIV-\naive) by processing leukoreduction filters as previously described \citep{Weitkamp:2001vm}. Briefly, RC2D leukoreduction filters were obtained from the American Red Cross and were backwashed with 35 mL of sterile PBS with 10mM EDTA. The resulting PBMC suspension was overlaid onto 15 mL of HistoPaque 1077 and centrifuged at 600 RCF for 25 minutes. The buffy coat was removed and washed twice with fresh PBS with 10mM EDTA. Total RNA was isolated from 10 million PBMCs using the RNeasy kit according to the manufacturer's standard operating procedure.

\subsubsection{cDNA Synthesis, PCR Amplification and Purification}
cDNA was synthesized from 100 ng of total RNA and 10 pmol of each RT-PCR Illumina-adapter primers in duplicate 50 \microliter~RT-PCR reactions using the OneStep RT-PCR system. The RT-PCR reactions were performed in a BioRad DNA Engine PTC-0200 thermal cycler running the following protocol: 50\degree C for 30:00, 95\degree C for 15:00, 35 cycles of (94\degree C for 0:45, 58\degree C for 0:45, 72\degree C for 2:00), 72\degree C for 10:00. cDNA synthesis was confirmed on a 1\% E-Gel EX. After which duplicate reactions were pooled. 2 \microliter~of each cDNA sample and 20 pmol of each indexed Illumina-adapter primer were used to template 100 \microliter~PCR amplification reactions in duplicate using the AmpliTaq Gold polymerase system. Thermal cycling was performed using the following protocol:Â 95\degree C for 10:00, 10 cycles of (95\degree C for 0:30, 58\degree C for 0:45, 72\degree C for 2:00), 72\degree C for 10:00. Amplicons were purified from the PCR reaction mix using the Agencourt AMPure XP system following the standard protocol, and duplicate reactions were pooled during the final elution. The removal of primers and correct amplicon size was confirmed on a 1\% E-Gel EX. Each amplicon sample was quantified using a Qubit fluorometer and the Quant-iT\textsuperscript{\textregistered} dsDNA HS Assay Kit and 8 indexed amplicon samples were pooled for each of the 8 lanes on the Illumina HiSeq flowcell.

\subsubsection{Illumina HiSeq Protocol}
The amplicon libraries underwent quality control by running on the Agilent Bioanalyzer High Sensitivity DNA assay to confirm the final library size and on the Agilent Mx3005P qPCR machine using the KAPA Illumina library quantification kit to determine concentration. For each library a 2 nM stock was created and denatured with NaOH. 12 pM of denatured libraries were loaded on the Illumina cBot for cluster generation on a paired-end flow cell. The flow cell was then loaded onto the Illumina HiSeq 2000 utilizing v3 chemistry and HTA 1.8. The raw sequencing reads in BCL format were processed through CASAVA-1.8.2 for FASTQ conversion and demultiplexing. The RTA chastity filter was used and only the pass filter reads were retained for further analysis.

\subsubsection{Paired-End Read Assembly and Junction Analysis}
FASTQ paired end reads were input into PANDAseq assembler software to produce a single sequence that was indexed by donor and position \citep{Bartram:2011cz}. Each sequence was uploaded to a custom database using the MongoDB framework that carried donor, position, sequence, and Phred quality score. The resulting sequences were concatenated and converted to FASTA format using BioPython SeqIO module \citep{Cock:2009hj}. Heavy chain CDR3 (HCDR3) junctions were analyzed using custom software. The software was modified to run in parallel on a high throughput computing cluster and to condense output to a minimum number of fields. The software was also modified to output the junction results in JSON format. The sequences were analyzed with BioPython to remove sequence ambiguity in each donor. The JSON files were then uploaded to the custom database using MongoDB framework. The two databases were linked by their donor id and position.

\subsubsection{30 Length HCDR3 Selection and Position Specific Structure Scoring Matrix (P3SM) Generation}
The custom database was queried for 30-length HCDR3 amino acid sequences generating > 26,000 unique sequences. 4,000 random sequences were selected for the pilot analysis in order to generate a custom position specific structure score matrix (P3SM) for PG9 HCDR3 structure. PG9 in complex with scaffolded template CAP45 (PDB ID: 3U4E) was used as a starting structure. The structure was stripped of waters and heavy chain and light chain constant regions. For the first round pilot, I also removed the CAP45 complex. Next, I used R\textsc{osetta}S\textsc{cripts} application available with the software suite from the R\textsc{osetta}C\textsc{ommons} (www.rosettacommongs.org) to thread and minimize the random HCDR3 sequences from HIV-\naive donors \citep{Fleishman:2011ji}. 50 decoys of each sequence were allowed to energetically minimize after threading yielding 200,000 models. 2,000 sequences (100,000 models) were used to fill the 30 by 20 P3SM using \rosetta per amino-acid energies of the HCDR3 loop. The remaining 2,000 sequences were used to benchmark the P3SM protocol.

\subsubsection{Selecting Sequences from the P3SM Heuristic for Validation}
After benchmark validation, the random 4,000 sequences were used in a final construction of a P3SM. Rapid prediction of score for each of the 26,000 HIV-\naive~HCDR3 sequences were calculated using the P3SM. PG9's sequence scored 112\textsuperscript{nd} out of 26,000 giving a noise tolerance of -3.82 REU (The top scoring sequence subtracted from the PG9 Score). Using $\pm$ 3.82 as my noise tolerance from PG9's score, 1,000 candidate sequences were selected to be further evaluated in complex.

\subsubsection{Sequence Tolerance Evaluated by Rosetta Design in Complex}
The top 1,000 candidate sequences evaluated by the P3SM were carried on to a separate \rosetta~protocol. This protocol evaluated sequence tolerance in complex with CAP45 antigen and surrounding glycans. N-linked glycan 156 and 160 (HXBC2 numbering) were both included in the complex input to \rosetta~as a non-canonical amino acid using the method described in Renfrew et al. \citep{Renfrew:2012ci}. After determining proper binding orientation with PG9, the entire complex was threaded with HIV-\naive~sequences. High-resolution docking perturbations were allowed but highly constrained to initial orientation using standard \rosetta~constraints files. I generated 100 models for each \naive~sequence and calculated a binding energy for each complex as:

\begin{gather*}
    \Delta \Delta \textup{G} = \Delta \textup{G}_{\textup{Bound}} - \Delta \textup{G}_{\textup{Unbound}} \\
    \textup{were,} \\
    \Delta \textup{G}_{\textup{Bound}} = \textup{RosettaScore}_{\textup{Complex}} \\
    \textup{and}\\
    \Delta \textup{G}_{\textup{Unbound}} = \textup{RosettaScore}_{\textup{Separated}}
\end{gather*}

In addition, the protocol was run a second time with sulfated tyrosines at positions 100G and 100H (Kabat numbering) if a tyrosine appeared at those positions in the HIV-\naive~sequences. Complex energies and interface binding metrics were parsed into a MySQL database for further analysis using included scripts from BioPython.

\subsubsection{Bootstrapping with Complex Energies}
The energy of each model evaluated in complex was reapplied to the P3SM and again ran through each HIV-\naive~donor sequence to predict a Rosetta energy using the same methodology as described. The bootstrapped models were included in the rest of the protocol.

\subsubsection{HIV-Na\"{i}ve Complex Energy Evaluation}
To filter \naive sequences into a realistic number to synthesize I evaluated multiple metrics. To weight each sequence, Z-Scores were assigned for the following score term metrics. HCDR3 total energy, HCDR3 C$\alpha$-RMSD, HCDR3 \ddg (the contribution to binding energy from just the HCDR3), total \ddg, ASN156 \ddg, and ASN158 \ddg. The Z-score is a measure of how many standard deviations a scoring metric fell from the mean. In terms of energy, all negative Z-Scores are preferred. When a Z-score was assigned for each HIV-\naive complex sequence, an average weighted Z-score was calculated using the following equation:

\begin{gather*}
\textup{Weighted-Z} = \frac{\sum_{i}^{N}w_{i}\times Z_{i}}{N}
\end{gather*}

Weights (w\textsubscript{i}) for each score term in the equation: total \ddg~-3.0, HCDR3 C$\alpha$-RMSD - 0.5, HCDR3-\ddg~-1.0, HCDR3 Score - 1.0, ASN156 \ddg~- 0.5, and ASN158 \ddg~- 0.5. This comprehensive metric can be used to rank-order each complex. In addition I used PG9 as a positive control and determined how many standard deviations away each of the HIV-\naive~complex scoring terms were from PG9's score using the following equation:
\begin{gather*}
\textup{Compare Score} = \frac{\bar{X}_{Scoring Term}-\bar{X}_{PG9 Scoring Term}}{\sigma_{PG9 Scoring Term}}
\end{gather*}

The compare score can then be weighted using the previous equation using the same weights to give one comprehensive metric to rank-order each HIV-\naive~sequence. The top 50 sequences were selected based on the average of the weighted compare score and weighted Z-score. 32 additional models were included from the bootstrapped protocol in the final results yielding 82 candidate HIV-\naive~sequences for experimental characterization.

\subsubsection{Clustering Analysis}
The sequences were clustered with ClustalW2 built in clustering algorithm after a multiple sequence alignment. The ClustalW plugin was used from the Genious Software suite (\url{http://www.geneious.com/}). The dendrogram was manually inspected and clusters were assigned yielding 10 candidate sequence groups for experimental characterization.

\subsubsection{Design Analysis for Sequence Tolerance}
Using the \rosettadesign~algorithm, the HIV-\naive~sequences tested for recovery using a small energetic bonus for favoring the native sequence \citep{Kuhlman:2000tc}. I applied a filter to minimize score and binding energy while favoring the native sequence. 100 models were generated using this protocol. After analysis, the sequence recovery was added to the Z-score metrics and the compare score using a weight of -2.0 (negative weight for favoring positive deviations) and reevaluated. Within each cluster, the HIV-\naive~sequence with the highest recovery and lowest Z-Score was further evaluated. For each mutated position, if a mutation was seen in greater that 10\% of the models and gave an energetic bonus of greater than 1.5 \rosetta~Energy Units (REUs), it was manually inspected using PyMOL and compared with the native sequence along with the native PG9 sequence from the native crystal complex (PDB ID: 3U4E).

\subsubsection{Antibody Expression}
To prepare HIV-\naive~PG9 variants and PG9 variant point mutations, I used recombinant expression in mammalian cells as previously described \citep{Xu:2010da}. Briefly, the MAb PG9 heavy- and light-chain genes were cloned into the pEE6.4 and pEE12.4 vectors, respectively. A BsiWI and XhoI cloning site were generated at AA position 95 and 110 (Kabat numbering), respectively. Using the unique cloning sites, the HIV-\naive~HCDR3 sequences were synthesized and cloned into the PG9 backbone. The DNA was co-transfected at a 1:1 heavy-light chain ratio into HEK 293F using polyethylenimine transfection reagent at a ratio 2:1 of PEI to DNA. 30 mL of culture was used for each variant and supernatant was collected on day 3.

CAP45 gp120 was cloned into pCNA3.4 using HindIII and EcoRI restriction sites. A CD5 signal peptide and 8X HIS tag was cloned onto the 5$'$ and 3$'$ end respectively. The DNA was transfected into HEK293F using polyethylenimine at a ratio of 2:1. On day 7, the supernatant was collected and purified with a 5 mL Talon cobalt HIS affinity column according to the manufactures specifications. The protein was concentrated using centrifugal units with a 100 kD cutoff.

\subsubsection{PG9/HIV Na\"ive Variant Antiboy Characterization}
ELISA plates were coated with 2 $\mu$g/mL of goat-anti-human (H+L) unlabeled antibody in PBS Buffer at 4\degree~overnight. The wells were washed with 0.05\% Tween and PBS Buffer all of the following steps. Using 2\% powdered milk and 1\% goat serum, the wells were blocked for 2 hours at room temperature. 200 \microliter~of supernatant collected from expression were applied to each well and allowed to complex with the capture antibody for 1 hour at 37\degree. Starting at 25 $\mu$g/mL, 100 \microliter~CAP45 gp120 was serially diluted at 1:3 in duplicate and allowed to bind for 1 hour at 37\degree. 100 \microliter~of mAb b12 was used diluted at 1 $\mu$g/mL in blocking buffer and allowed to incubate for 1 hour at 37\degree. 100 \microliter~of 1:5,000 of goat-anti-human labeled with horseradish peroxidase secondary was added to each well and allowed to incubate for 1 hour at 37\degree. 100 \microliter~of 3,3$'$,5,5$'$-tetramethylbenzidine was added to each well. The reaction was stopped with 1N HCL and read at 450 nM absorbance. The \ec~of each HIV-\naive~variant was compared with PG9 positive control.


\subsubsection{Statistics and Graph Generation}
All statistics were calculated in the R-programming language (\url{http://www.r-project.org}) or GraphPad package. All graphs were generated in GraphPad package or the ggplot2 library (\url{http://ggplot2.org}) in the R-programming language.

\clearpage

\subsection{Chapter \ref{chap:chapter4} - Materials and Methods}
\par\vspace{10pt}
\subsubsection{Position Specific Scoring Matrix to Determine the Tolerance of Diverse Sequences to the Hammerhead Structure of PG9}
We obtained large numbers of human PBMCs from 64 otherwise healthy HIV-negative subjects by recovering cells from leuko-reduction filters obtained from the Nashville, TN Red Cross. Bryan Briney extracted total RNA from white blood cells retained in the filters, then performed RT-PCR amplification of expressed antibody heavy chain genes using primers designed to amplify all human heavy chain antibody sequences \citep{Briney:2012ib}. I determined the sequences of the HCDR3 region of the amplicons using HiSeq next generation sequencing (Illumina) according to the manufacture's instructions. Amplifying and sequencing 64 donors separately yielded a total of $5.14~x~10^{8}$ HCDR3 sequences. A subset of 4,000 randomly selected 30-amino acid length HCDR3 sequences was used to determine what amino acids were tolerated by antibodies in the hammerhead configuration of the PG9\textit{wt} HCDR3 by threading each sequence over the backbone coordinates of PG9\textit{wt} using \rosetta. The backbone was energetically minimized with iterative rounds of small docking perturbations. Scores of each amino acid were input into a custom position specific scoring matrix (PSSM). The matrix then was used to rapidly compute the remaining 30 length amino acids predicted score given by \rosetta.

\subsubsection{Redesign of PG9 HCDR3}
Using the \rosettadesign~algorithm, iterative rounds of design, docking, and minimization were applied to each position in the HCDR3 with a small energetic bonus applied to recovery of the native sequence \citep{Kuhlman:2000tc}. 100 models were generated using this protocol (see protocol capture). For each mutated position, if a mutation was seen in greater that 10\% of the models and gave an energetic bonus of greater than 1.0 \rosetta~energy Units, it was manually inspected using PyMOL and compared with the native sequence along with the native PG9 sequence from the native crystal complex (PDB ID-3U4E) \citep{McLellan:2011dg}.


\subsubsection{Antibody and gp120 Expression}
To prepare HIV-\naive~PG9 variants and PG9 variant point mutations, I used recombinant expression in mammalian cells as previously described \citep{Xu:2010da}. Briefly, the mAb PG9 heavy- and light-chain genes were cloned into the pEE6.4 and pEE12.4 vectors, respectively (Lonza). BsiWI and XhoI cloning sites were generated at AA position 95 and 130, respectively. HIV-\naive~HCDR3 sequences were synthesized, and cloned into the PG9 backbone (GeneArt) using the unique cloning sites. The DNA was co-transfected at a 1:1 heavy-light ratio into FreeStyle 293-F cells (Life Technologies) using 25 kDa linear polyethylenimine (PEI, Polysciences Inc.) transfection reagent at a ratio 2:1 of PEI to DNA. 30 mL of culture was used for each variant and supernatant was collected on day 5 and purified on a protein G column (GE).

Each gp120 was cloned into pCDNA3.4 (Life Technologies) using HindIII and EcoRI restriction sites. A CD5-signal peptide and 8x His tag was cloned onto the 5' and 3' end, respectively. The DNA was transfected into FreeStyle 293-F cells using PEI at a ratio of 2:1 (Life Sciences). On day 5, the supernatant was clarified and the protein purified on a 5 mL HisTALON cobalt column (Clontech) according to the manufacturers specifications. The protein was concentrated using Amicon Ultra centrifugal filters with a 100 kD cutoff (Millipore, Billerica, MA) and further purified on a Superdex column (GE) using size exclusion. BG505 SOSIP.664 trimer was received as a gift from John Moore.

\subsubsection{PG9 Variant Characterization}
ELISA plates were coated with 3 $\mu$g/mL of gp120 and incubated overnight at 4\degree C. The wells were washed with phosphate buffered saline with 0.05\% Tween (PBS-T) in all of the following steps. The uncoated sites on the wells were blocked with 2\% skim milk and 1\% goat serum in PBS-T for 2 hours at room temperature.  All antibodies were diluted serially in two-fold starting from 25 $\mu$g/mL for 24 dilutions. Horseradish peroxidase-conjugated goat-anti-human IgG was added to each well and allowed to incubate for 1 hour at 37\degree C and color developed with 3,3,5$'$,5$'$-tetramethylbenzidine (Thermo). The reaction was stopped with 1N HCl and read at 450 nM. The EC\textsubscript{50} of each PG9 variant was compared with PG9 positive control.

For BG505 SOSIP.664 Trimer, ELISAs were performed according the protocol as previously described \citep{Sanders:2013gm}. Maxisorp 96-well plates (Nunc) were coated overnight with mAb D7324 (Aalto Bioreagents) at 5 mg/mL in 0.1 M NaHCO\textsubscript{3}, pH 8.6 (100 $\mu$L/well). After the washing and blocking steps, purified, D7324-tagged BG505 Env proteins were added at 800 ng/mL in PBS and 2\% milk for 2 h at ambient temperature and the unbound Env proteins were washed away. PG9 and PG9-variants were diluted to 25 $\mu$g/mL in PBS with 10\% sheep serum/2\% milk and diluted serially 2-fold and allowed to incubate for 2 h at room temperature followed by 3 washes with PBS-T. Horseradish peroxidase-conjugated goat-anti-human IgG was added for 1 h at a 1:3,000 dilution (final concentration 0.33 mg/mL) in 10\% sheep serum/2\% milk, followed by 5 washes with PBS-T. Color development and optical density measurement was done as above.

\subsubsection{Neutralization Assays}
Neutralization was measured as a function of reductions in luciferase (Luc) reporter gene expression after a single round of infection in TZM-bl cells as described \citep{Montefiori:2009hj,Simek:2009cn}. This assay has been formally optimized and validated and was performed in compliance with Good Clinical Laboratory Practices \citep{SarzottiKelsoe:2013hr}. TZM-bl cells were obtained from the NIH AIDS Research and Reference Reagent Program, as contributed by John Kappes and Xiaoyun Wu.  Briefly, virus at a dose of 50,000-150,000 relative luminescence units (RLU) equivalents was incubated with serial 3-fold dilutions of test sample in duplicate in a total volume of 150 \microliter~for 1 hr at 37\degree C in 96-well flat-bottom culture plates.  Freshly trypsinized cells (10,000 cells in 100 \microliter~of growth medium containing 75 $\mu$g/mL DEAE dextran) were added to each well.  One set of control wells received cells + virus (virus control) and another set received cells only (background control). After a 48 hour incubation, 100 \microliter~of cells was transferred to a 96-well black solid plates (Costar) for measurements of luminescence using the Britelite Luminescence Reporter Gene Assay System (PerkinElmer Life Sciences). Neutralization titers are the dilution at which RLU were reduced by 50\% compared to virus control wells after subtraction of background RLUs.  Assay stocks of molecularly cloned Env-pseudotyped viruses were prepared by transfection in 293T cells and were titrated in TZM-bl cells as described \citep{Li:2005go}. Additional details of the assay and all supporting protocols may be found at \url{http://www.hiv.lanl.gov/content/nab-reference-strains/html/home.htm}.


All of the Env-pseudotyped viruses used for these assays exhibited a Tier 2 neutralization phenotype except for TH023.6 and TH023.6/N160A.5, which exhibited a tier 1A phenotype. The Envs for these pseudoviruses were derived from genetic subtypes A (398\_F1\_F5\_20), B (WITO4160.33, X2278\_C2\_B6, SC422661.8, TRO.11, SC22.3C2.LucR.T2A.ecto), C (Ce703010217, Du422.1, Ce1086\_B2), G (X1632\_S2\_B6) and CRF01\_AE (CNE55, R2184.c04).

\subsubsection{Statistics and Graph Generation}
All statistics were calculated in the R-programming language (\url{http://www.r-project.org}) or Prism package (GraphPad) through the Ipython interface \url{www.ipython.org}. All graphs were generated in Prism package or the ggplot2 library (\url{http://ggplot2.org}) in the R-programming language.
\clearpage