\section{Appendix I - \rosetta~Glossary}
\label{sec:appI}
\singlespace
\setlength{\parindent}{0pt}

\textbf{All-atom} - in the case of sampling, synonymous with fine movements and often including side chain information; also referred to as high-resolution \\ \\

\textbf{Benchmark} - another word for a test of a method, scoring function, algorithm, etc. by comparing results from the method to accepted methods/models \\ \\

\textbf{Binary file} - a file in machine-readable language that can be executed \textit{in silico} \\ \\

\textbf{BioPython} - a set of tools for biological computing written and compatible with Python \\ \\

\textbf{Build} - to compile the source code so it may be used as a program \\ \\

\textbf{Centroid} - in \rosetta centroid mode, side chains are represented as unified spheres centered at the residues center of mass \\ \\

\textbf{Cluster center} - the geometric center of a cluster, or group, of models \\ \\

\textbf{Clustering} - in this case, grouping models with similar structure together \\ \\

\textbf{Comparative model} - a protein model where the primary sequence from one protein (target) is placed, or threaded, onto the three dimensional coordinates of a protein of known structure (template) \\ \\

\textbf{Cyclic coordinate descent (CCD)} - based on robotics, CCD loop closure is used to build loops in \rosetta by fragment assembly and close loops by decreasing the gap between two termini in three-dimensional space \\ \\

\textbf{\textit{De novo}} - from the sequence; also called \textit{ab initio}, with no experimental guidance \\ \\

\textbf{Directory} - synonymous with a folder, usually contains one or more files or other folders \\ \\

\textbf{distance matrix} - a matrix containing the pairwise distances for every point in a set of points \\ \\

\textbf{Dunbrack rotamer library} - a set of likely side chain conformations for the twenty canonical amino acids based on protein structures in the Protein Data Bank (PDB) \\ \\

\textbf{Executable} - binary file used to execute the program \\ \\

\textbf{Force field/Scoring function/Energy function/Potential} - often used interchangeably; a means of assessing the energy of the generated models \\ \\

\textbf{Fragment} - in \rosetta folding and loop building, a set of three-dimensional coordinates corresponding to a given amino acid sequence fragment \\ \\

\textbf{Database} - also called the fragment library, contains all the interchangeable data needed for \rosetta \\ \\

\textbf{Gap} - in sequence alignment, a gap is inserted when the sequences are of low homology; usually appear as a dash (-); the gaps form a sequence alignment correspond to areas where loops are
built during comparative modeling \\ \\

\textbf{GDT/GDT\_TS} - global distance test (total score); a measure of similarity between two protein structures having the same amino acid sequence; the largest set of residues C$\alpha$ atoms in the model structure falling within a defined distance cutoff of their position in the experimental structure \\ \\

\textbf{Gradient-based minimization} - also known as minimization by steepest descent; in this case, a means of energy minimization in which one takes steps proportional to the negative of the gradient of the function (energy) at the current point \\ \\

\textbf{High-resolution} - in the case of sampling, synonymous with fine movements and often including side chain information \\ \\

\textbf{Homology model} - a more specific type of comparative model where the protein sequence of interest (target) is a homolog of the protein of known structure (template) \\ \\

\textbf{Interface delta} - the interface delta score is defined as the contribution to the total score for which the presence of the ligand is responsible \\ \\

\textbf{Kinematic loop closure (KIC)} - robotics-inspired loop closure algorithm which analytically determines all mechanically accessible conformations for torsion angles of a peptide chain using polynomial resultants \\ \\

\textbf{Knowledge-based} - in the case of \rosetta, based on information obtained from structures found in the PDB \\ \\

\textbf{Libraries} - in computing, a collection of code and data (classes and functions) used by a piece of software and is often used in software development \\ \\

\textbf{Ligand} - the part of the structure that binds to a protein to serve some biological purpose \\ \\

\textbf{Low-resolution} - a somewhat subjective term, in the case of sampling, synonymous with coarse movements of the protein and/or ligand backbone and side chains; the individual atoms of low-resolution structures or models cannot be resolved, or observed \\ \\

\textbf{Metropolis criterion} - often combined with the Monte Carlo sampling algorithm; allows for generation of an ensemble that represents a probability distribution \\ \\

\textbf{Model} - in the case of this protocol, a structure generated by \rosetta; sometimes called a decoy \\ \\

\textbf{Monte Carlo sampling} - a randomized and repetitive computational sampling method \\ \\

\textbf{Mover} - a generic class that takes as input a pose and performs some modification on that pose; for example, a mover might take in a pose and rotate every residue \\ \\

\textbf{Namespace} - in computer science, an abstract container holding a logical grouping of unique identifiers or symbols; in \rosetta, examples of namespaces are loops, relax, etc. \\ \\

\textbf{Native-like} - close to the experimentally determined structure; a model that is native-like usually has an RMSD to the experimentally determined structure of < 2\r{A} \\ \\

\textbf{Options file} - often called a flags file; a file containing \rosetta options that can be passed to a \rosetta executable after the @ symbol; can be easier to use than passing \rosetta options over the command line \\ \\

\textbf{Pack/repack} - in \rosetta, side chains are packed/repacked by switching out rotamers and scoring them using the \rosetta scoring function \\ \\

\textbf{Pose} - in  \rosetta protocol, a three-dimensional conformation of the ligand, protein, or ligand/protein complex at any given time-point \\ \\

\textbf{Python} - interpreted, object-oriented, high-level programming language \url{http://www.python.org/} \\ \\

\textbf{Relax} - in \rosetta, an iterative protocol of side chain repacking and gradient-based minimization; often referred to as full-atom (or all-atom) refinement \\ \\

\textbf{Robetta}  \rosetta structure prediction server (\url{http://robetta.bakerlab.org/}) freely available to not-for-profit users \\ \\

\textbf{RosettaCommons} - a group of more than twenty labs that develop the \rosetta software suite  \\ \\

\textbf{REU} - arbitrary energy units specific to the \rosetta scoring function \\ \\

\textbf{RosettaScripts} - also called ``the scripter'' or RosettaXML; an XML-like language that allows for specifying modeling tasks in \rosetta \\ \\

\textbf{Rotamer} - rotational conformer of an amino acid or ligand side chain \\ \\

\textbf{SCons} - a tool for constructing software from its source code \url{http://www.scons.org/} \\ \\

\textbf{Script} - in computer programming, a script is a sequence of instructions that is interpreted or carried out by another program rather than by the computer processor (as a compiled program is) \\ \\

\textbf{Source code} - human-readable files that are the implementation of the program; are written in C++ in \rosetta \\ \\

\textbf{Target} - in comparative, or homology, modeling, the protein for which we are generating a model; the target sequence is the primary sequence of the protein for which we want to make a model \\ \\

\textbf{Template} - in comparative modeling, the protein of known structure on which the target is threaded \\ \\

\textbf{Threading} - placing the primary sequence of one protein (target) on the three-dimensional coordinates of a protein of known structure (template) based on a sequence alignment loop building \\ \\

\textbf{XML} - Extensible Markup Language; in this case, used to write protocols to pass to

\clearpage